app/controllers/game.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Module dependencies.
+ */
+const mongoose = require('mongoose'),
+  Game = mongoose.model('Game');
+
+/**
+ * Save Game
+ */
+exports.saveGame = (req, res) => {
+  if (req.body.gameId) {
+    Game.findOneAndUpdate({ gameId: req.body.gameId })
```

Comment:
`Game.findOneAndUpdate` isn't doing exactly doing what you think it is, please read more about the method. However, you need to use `Game.find` and if the `game` doesn't exist || there's an `err` then you `throw`. If not you go ahead and save the `game` like you're doing on line 20. 

@Daramola98